test/specs/EventsSpec.js

```diff
@@ -224,6 +224,16 @@ describe("Event Tests", function() {
 
     });
 
+    describe("Window Events", function() {
+      it("'resize' event is triggered properly and can be binded to", function() {
+        testSlider.on('resize', function() {
+          flag = true;
+        });
+        testSlider.data('slider')._resize();
```

Comment:
Also,  since this fix is intended to fix the positioning of the ticks, could you add some assertions to the test to ensure the ticks are positioned correctly after the `resize` event is triggered on the `window`?